pkg/services/thumbs/datasources_lookup_test.go

```diff
@@ -0,0 +1,57 @@
+package thumbs
+
+import (
+	"context"
+	_ "embed"
+	"testing"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+	"github.com/grafana/grafana/pkg/services/searchV2"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	//go:embed testdata/search_response_frame.json
```

Comment:
Ha! that's cool :)

Reply:
Emil taught me :D https://github.com/grafana/grafana/pull/51805#discussion_r914807817